django_project/base/templates/landing_page.html

```diff
@@ -15,7 +15,7 @@
 {% block body_content %}
 <div id="section-wrapper">
     <header class="masthead text-center text-white d-flex" id="header">
-        {% include "carousel.html" %}
+{#        {% include "carousel.html" %}#}
```

Comment:
Why did you comment this?

Reply:
I have uncommented it, it was because I was using an outdated django-bims install which did not hav e carousel yet. After updating the package its fine.

Reply:
Its still there if it shows here @sonlinux